examples/demo/src/routes/HomePage.tsx

```diff
@@ -24,9 +23,10 @@ export const HomePage: React.FC = () => {
   ] = useApplicationContext();
 
   return (
-    <Recommend recommendClient={recommendClient}>
+    <>
```

Comment:
Removed the Recommend context here (but is kept in `ProductPage` to showcase the usage with and without.